readme.md

```diff
@@ -84,6 +85,7 @@ Configure it in `package.json`.
 - [custom-error-definition](docs/rules/custom-error-definition.md) - Enforce correct `Error` subclassing. *(fixable)*
 - [error-message](docs/rules/error-message.md) - Enforce passing a `message` value when throwing a built-in error.
 - [escape-case](docs/rules/escape-case.md) - Require escape sequences to use uppercase values. *(fixable)*
+- [expiring-todo-comments](docs/rules/expiring-todo-comments.md) - Add conditions to TODO comments to make them error.
```

Comment:
This should be in sync with the docs title.

Reply:
Done